tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -249,21 +241,24 @@ Bridge network - ESX none
     Cleanup VIC Appliance On Test Server
 
 Bridge network - create bridge network if it doesn't exist
-    ${status}=  Get State Of Github Issue  5293
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 6-07-Create-Network.robot needs to be updated now that Issue #5293 has been resolved
-    #Run Keyword If  '%{HOST_TYPE}' != 'ESXi'  Pass Execution  Test skipped on vCenter
+    Run Keyword If  '%{HOST_TYPE}' != 'ESXi'  Pass Execution  Test not applicable on vCenter
+    # ESX should automatically create the bridge network AAAAAAAAAA, but vCenter would fail with unknown network error
+    Set Test Environment Variables
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
 
-    #Set Test Environment Variables
-    ## Attempt to cleanup old/canceled tests
-    #Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
-    #Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
```

Comment:
@morris-jason, are lines 251, 252, and 253 repeats of lines 247, 248, and 249?  Is there a reason you want to cleanup old/canceled tests twice?  Curious about this.